ui/apps/platform/src/Containers/AccessControl/AccessScopes/AccessScopes.tsx

```diff
@@ -17,7 +17,6 @@ import {
     accessScopeNew,
     createAccessScope,
     deleteAccessScope,
-    getIsDefaultAccessScopeId,
```

Comment:
For extra credit, can you please add a comment at the definition of `getIsDefaultAccessScopeId` function that the only remaining import is in ResourceScopeSelection.tsx file, which will be deleted when Collections (which depend on postgres) supersede Access Scopes in Vulnerability Reporting.